modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcast/PodcastAutoArchiveFragment.kt

```diff
@@ -100,20 +103,52 @@ class PodcastAutoArchiveFragment : PreferenceFragmentCompat() {
         }
 
         preferenceCustomForPodcast?.setOnPreferenceChangeListener { _, newValue ->
-            viewModel.updateGlobalOverride(newValue as Boolean)
+            analyticsTracker.track(
+                AnalyticsEvent.PODCAST_SETTINGS_AUTO_ARCHIVE_TOGGLED,
+                mapOf("enabled" to (newValue as Boolean))
+            )
+            viewModel.updateGlobalOverride(newValue)
             true
         }
 
         preferenceAutoArchivePodcastPlayedEpisodes?.setOnPreferenceChangeListener { _, newValue ->
             val stringVal = newValue as? String ?: return@setOnPreferenceChangeListener false
             val index = max(afterPlayingValues.indexOf(stringVal), 0) // Returns -1 on not found, default it to 0
+            analyticsTracker.track(
+                AnalyticsEvent.PODCAST_SETTINGS_AUTO_ARCHIVE_PLAYED_CHANGED,
+                mapOf(
+                    "value" to when (index) {
+                        0 -> "never"
+                        1 -> "after_playing"
+                        2 -> "after_24_hours"
+                        3 -> "after_2_days"
+                        4 -> "after_1_week"
+                        else -> "unknown"
+                    }
```

Comment:
I didn't define constants for these kinds of property values  in this PR because it feels like doing that has just been causing us more trouble than it's worth (imho), so I wanted to try just using plain strings for the analytics props. Let me know what you think. I'm happy to pull these out into constants if you prefer.

Reply:
This is ok for me.